src/properties/struct_property.rs

```diff
@@ -176,6 +178,9 @@ impl StructProperty {
             "DateTime" => StructPropertyValue::DateTime(DateTime {
                 ticks: UInt64Property::read(cursor, false)?.value,
             }),
+            "Timespan" => StructPropertyValue::Timespan(DateTime {
```

Comment:
I would suggest making a Timestamp struct

Reply:
shoudn't this follow the same rules as DateTime?

Reply:
the representation is the exact same - internally they're both referred to as ticks so I guess I could maybe rename the struct to Time or smthn

Reply:
> shouldn't this follow the same rules as DateTime?

it needs a different property name which is why I didn't just | "Timespan"

Reply:
yeah this was a question to Scott